tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -167,6 +168,21 @@ def testing_config(request, duthosts, rand_one_dut_hostname):
         restart_dhcp_service(duthost)
 
 
+def _check_link_status(duthost, iface_list, expect_status):
```

Comment:
Let's check if such a function already exists in sonic-mgmt, if not- probably this one should be placed under a more generic location